### PR TITLE
BTC-Trigger

### PR DESCRIPTION
**Pull request by awisniew207**
Code example to match the blog post

### CODE REVIEW

File: btc-trigger/nodejs/README.md
Comment (spacesailor24) [inline btc-trigger/nodejs/README.md:1]: Do we want to provide a prerequisites section here?

File: btc-trigger/nodejs/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+import * as bip66 from "bip66";
+import * as crypto from "crypto";
+
+import { litActionCode } from "./litAction";
+
+const PKP_PUBLIC_KEY = process.env["PKP_PUBLIC_KEY"]!;
+const ETHEREUM_PRIVATE_KEY = process.env["ETHEREUM_PRIVATE_KEY"]!;
+const BTC_DESTINATION_ADDRESS = process.env["BTC_DESTINATION_ADDRESS"]!;
+const LIT_CAPACITY_CREDIT_TOKEN_ID =
+  process.env["LIT_CAPACITY_CREDIT_TOKEN_ID"];
```
Comment (spacesailor24) [inline btc-trigger/nodejs/src/index.ts:27]: Which of these are required? We should be using the `getEnv` util function for the required ones

File: btc-trigger/nodejs/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+const LIT_NETWORK = LitNetwork.DatilTest;
```
Comment (spacesailor24) [inline btc-trigger/nodejs/src/index.ts:29]: Lets also make this an ENV that defaults to Datil

File: btc-trigger/nodejs/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+  ETHEREUM_PRIVATE_KEY,
+  new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+);
+
+const address = ethersWallet.address;
+
+const EC = elliptic.ec;
+bitcoin.initEccLib(ecc);
+
+export const executeBtcSigning = async () => {
```
Comment (spacesailor24) [inline btc-trigger/nodejs/src/index.ts:41]: Can you wrap all the code here with `try/catch/finally` like our other code examples?

File: btc-trigger/nodejs/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+  tx.setInputScript(0, scriptSig);
+  console.log("✅ Input script set");
+
+  const txHex = tx.toHex();
+
+  console.log("🔄 Broadcasting transaction...");
+  const broadcastTransaction = async (txHex: string) => {
+    try {
+      const response = await fetch("https://mempool.space/api/tx", {
```
Comment (spacesailor24) [inline btc-trigger/nodejs/src/index.ts:236]: `https://mempool.space/api/tx` should be an ENV

File: btc-trigger/nodejs/src/index.ts (hunk 2 above)
Comment (awisniew207) [reply btc-trigger/nodejs/src/index.ts:29]: That gives these errors, not breaking, but might confuse people:

> Type 'string' is not assignable to type 'LIT_NETWORKS_KEYS'.ts(2322)
interfaces.d.ts(156, 5): The expected type comes from property 'litNetwork' which is declared here on type 'LitNodeClientConfig | CustomNetwork'
(property) litNetwork: string

Comment (spacesailor24) [reply btc-trigger/nodejs/src/index.ts:29]: I think you can just specify the type to get rid of the error

**Review by spacesailor24 — APPROVED** [review]

**Pull request merged**